### PR TITLE
Add a tutorial for using Earthly with Space ROS dev

### PR DESCRIPTION
**Pull request by nuclearsandwich**
This is still a work in progress and I want to add more text to cover some of the questions asked in an earlier demonstration but it's the first Space ROS-specific tutorial!

### CODE REVIEW

File: source/How-To-Guides/Reproducing-CI-Results.rst [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+
+Space ROS CI runs on a daily basis using GitHub Actions and builds all packages included in the `Space ROS repos file <https://github.com/space-ros/space-ros/blob/main/ros2.repos>`_.
+Two artifacts are produced from these builds: an OCI-compliant container image published to Docker Hub as ``osrf/space-ros``, and a `build results archive <https://github.com/space-ros/process_sarif/blob/main/docs/build-results-archive.md>`_ available as a GitHub Actions artifact.
+
+Prerequisites
+-------------
+
+In order to reproduce these artifacts locally, you will need the following:
+
+* `earthly installation <https://earthly.dev/get-earthly>`__ which requires `Docker <https://docker.com>`__ (recommended) or `Podman <https://podman.io>`__.
```
Comment (mjeronimo) [inline source/How-To-Guides/Reproducing-CI-Results.rst]: Even though it is an option, technically, should we call out Podman? Perhaps it is in the "possible, but unsupported" category.

File: source/Tutorials.rst [hunk 2]
```diff
@@ -5,6 +5,7 @@ Tutorials
 
 If you're new to ROS 2, we recommend first reviewing the `ROS 2 tutorials <https://docs.ros.org/en/humble/Tutorials.html>`_, which are a collection tutorials with step-by-step instructions meant to steadily build skills in ROS 2.
 
+<<<<<<< HEAD
```
Comment (mjeronimo) [inline source/Tutorials.rst]: A merge artifact

File: source/Tutorials/First-Time-Building.rst [hunk 3]
```diff
@@ -1,6 +1,102 @@
...
+* `earthly installation <https://earthly.dev/get-earthly>`__ which requires `Docker <https://docker.com>`__ (recommended) or `Podman <https://podman.io>`__.
+* Understanding of basic terminal commands (`here’s a guide for Linux <http://www.ee.surrey.ac.uk/Teaching/Unix/>`__)
+
+Introduction
+------------
+
+Space ROS, like ROS 2, is composed of a number of independent software projects each maintained in its own repositories.
+The ROS community has a number of tools to help organize and improve working with multiple repositories.
+Earthly is an automation tool where each target is run in a dedicated container with the ability to share artifacts between targets and the local host.
+The Space ROS project primarily uses earthly to create automation that runs consistently in local development, and in hosted infrastructure such as GitHub Actions.
```
Comment (mjeronimo) [inline source/Tutorials/First-Time-Building.rst]: earthly -> "Earthly"

Comment (mjeronimo) [reply source/Tutorials/First-Time-Building.rst]: Additional occurrences below in the text. 

Should consistently use "Earthly" when referring to the name of the tool in the abstract. "earthly" is OK when referring to the name of the binary.  This is my assumption, at least. I recommend following the conventions used in the Earthly docs.

File: source/Tutorials/First-Time-Building.rst [hunk 4]
```diff
@@ -1,6 +1,102 @@
...
+.. code-block:: console
+
+  earthly +sources
+
+4 Optionally update Space ROS sources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to check that Space ROS builds in its default configuration, you can jump to step 5 below and return after to verify that Space ROS's default configuration is currently building.
+
+At this point you can make any modifications or additions to the repositories in src before continuing with the build.
```
Comment (mjeronimo) [inline source/Tutorials/First-Time-Building.rst:81]: Might provide an example of why one might modify or add to the src directory before continuing with a build.

File: source/Tutorials/First-Time-Building.rst (hunk 3 above)
Comment (nuclearsandwich) [reply source/Tutorials/First-Time-Building.rst]: Yeah I agree that it should be consistent. Corrected in f4f1277a1d7573eeff21134b6fda5807ec1c489b

File: source/Tutorials/First-Time-Building.rst (hunk 4 above)
Comment (nuclearsandwich) [reply source/Tutorials/First-Time-Building.rst:81]: I added an additional sentence in 41792edeeb3962e4d7f32557cd6aad00ccdb4ac6 but I'm happy to keep workshopping it.

**Review by mjeronimo — APPROVED** [review]

File: source/How-To-Guides/Reproducing-CI-Results.rst (hunk 1 above)
Comment (nuclearsandwich) [reply source/How-To-Guides/Reproducing-CI-Results.rst]: Removed in subsequent changes. Anyone who knows podman as a viable alternative likely doesn't need to be told as much in this tutorial.

**Comment by nuclearsandwich** [comment]
I have had additional realtime approval from @mjeronimo and am merging the current state!

**Comment by nuclearsandwich** [comment]
I ended up reverting the link because I couldn't get the syntax right.

**Pull request merged**